src/api/app/models/binary_release.rb

```diff
@@ -187,7 +187,7 @@ def indentical_to?(binary_hash)
     binary_disturl == binary_hash['disturl'] &&
       binary_supportstatus == binary_hash['supportstatus'] &&
       binary_buildtime &&
-      binary_buildtimeto.datetime.utc == ::Time.at(binary['buildtime'].to_i).to_datetime.utc
+      binary_buildtime.datetime.utc == ::Time.at(binary['buildtime'].to_i).to_datetime.utc
```

Comment:
I think it is not a type it was meant to be ``binary_buildtime_TO.datetime.utc``
But anyway, I'm fine with the change!

Reply:
Nah, it should be to_datetime :/